webui/src/lib/components/repository/changes.jsx

```diff
@@ -143,24 +185,35 @@ export const TreeEntryRow = ({entry, relativeTo = "", leaf = false, dirExpanded,
         pathSection = <Link href={onNavigate(entry)}>{pathSection}</Link>
     }
     const rowActions = []
-    if (onClickExpandDiff) {
-        rowActions.push(new RowAction(<FileDiffIcon/>, diffExpanded ? "Hide changes" : "Show changes", diffExpanded, onClickExpandDiff))
-    }
-    if (!leaf) {
-        rowActions.push(new RowAction(<GraphIcon/>, showSummary ? "Hide summary" : "Calculate change summary", showSummary, () => setShowSummary(!showSummary)))
-    }
-    if (onRevert) {
-        rowActions.push(new RowAction(<HistoryIcon/>, "Revert changes", false, () => {
-            setShowRevertConfirm(true)
-        }))
+    switch (changesTreeType) {
+        case ChangesTreeType.ObjectChanges:
+            if (onClickExpandObjectDiff) {
+                rowActions.push(new RowAction(<FileDiffIcon/>, diffExpanded ? "Hide changes" : "Show changes", diffExpanded, onClickExpandObjectDiff))
+            }
+            if (!leaf) {
+                rowActions.push(new RowAction(<GraphIcon/>, showSummary ? "Hide summary" : "Calculate change summary", showSummary, () => setShowSummary(!showSummary)))
+            }
+            if (onRevert) {
+                rowActions.push(new RowAction(<HistoryIcon/>, "Revert changes", false, () => {
+                    setShowRevertConfirm(true)
+                }))
+            }
+            break;
+        case ChangesTreeType.DeltaLakeTableChanges:
+            if (isDeltaTableRootEntry) {
```

Comment:
Rows for Delta tables should be marked somehow. Can be one or more of:
1. Put the Delta logo somewhere.
2. Make them unclickable (at this point clicking them has no meaning anyway).
3. Lose the `/` at the end of them.

Reply:
> Make them unclickable (at this point clicking them has no meaning anyway).

Do you mean making the path unclickable? nice Idea I'll implement it!